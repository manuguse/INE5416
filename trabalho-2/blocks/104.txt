1 1 1 1 12 12 16 18 18 18
2 1 1 8 12 12 16 16 18 21
2 2 2 8 8 12 12 18 18 20
2 4 4 8 8 13 13 18 20 20
3 3 4 4 13 13 13 17 20 17
3 4 4 4 14 14 17 17 17 17
3 5 7 9 9 14 14 14 14 17
3 5 5 10 9 14 15 19 19 22
3 6 5 11 9 15 15 19 19 19
3 6 6 11 11 15 15 15 19 19