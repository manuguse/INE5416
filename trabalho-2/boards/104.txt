- - - 2 4 - 3 - - 3
3 - 5 - - - - 1 4 -
2 - - - - 2 1 - - 2
1 6 5 - 1 5 2 - - -
- - - - - - - 5 - 6
6 - 3 - 3 - - - - 4
- - - - - 2 4 - 7 2
4 - 2 - 2 - - 5 3 -
- - - 3 - 6 - - - -
- 1 - 1 - - - 5 - -