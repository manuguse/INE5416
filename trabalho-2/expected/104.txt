4 3 6 2 4 6 3 2 5 3
3 1 5 4 3 5 2 1 4 1
2 5 4 3 5 2 1 7 1 2
1 6 5 2 1 5 2 6 4 1
7 1 4 7 4 3 1 5 3 6
6 2 3 1 3 5 7 3 1 4
5 4 1 3 4 2 4 6 7 2
4 3 2 1 2 1 3 5 3 1
3 2 1 3 1 6 2 4 2 7
2 1 3 1 2 4 1 5 1 6